_includes/nav.html

```diff
@@ -1,23 +1,21 @@
-<!-- <nav class="">
-  <div class="container">
-
-
-
-
-    <a href="/" {% if page.url == "/" %}class="active"{% endif %}>Home</a>
-    |
-    <a href="/about" {% if page.url == "/about" %}class="active"{% endif %}>About</a>
-
-  </div>
-</nav> -->
+<<nav id="nav"  class="navbar navbar-expand-sm ">
+      <div class="container nav__container">
 
+        {% if page.url contains "bcjobs" %}
+          <a class="navbar-brand" href="https://www.bcjobs.ca/">
+        {% include svg-logo.html %}
+        </a>
```

Comment:
There's a bug here. Closing `</a>` is not enclosed inside the if statement, which leads to this:

![image](https://user-images.githubusercontent.com/504505/53317601-28535f80-3881-11e9-89a0-63bb7c5b1162.png)